video/out/vo_gpu_next.c

```diff
@@ -964,9 +964,11 @@ static void draw_frame(struct vo *vo, struct vo_frame *frame)
         );
 
         // mpv likes to generate sporadically jumping PTS shortly after
-        // initialization, but pl_queue does not like these. Hard-clamp as
-        // a simple work-around.
-        qparams.pts = p->last_pts = MPMAX(qparams.pts, p->last_pts);
+        // initialization, but pl_queue does not like these. Hard-clamp to
+        // the first frame in the queue as a simple workaround.
+        struct pl_source_frame first;
+        if (pl_queue_peek(p->queue, 0, &first))
+            qparams.pts = MPMAX(qparams.pts, first.pts);
```

Comment:
Can we have at least MP_VERBOSE warnings when this clamping happens? We probably should smooth those jump and this will make clear in log that something is not working.

Reply:
Sure, added a message.